user_tools/docs/user-tools-onprem.md

```diff
@@ -219,4 +219,95 @@ The command creates a directory with UUID that contains the following:
+| **eventlogs**        | A comma separated list to event logs or directory                                                                                                                                                                                                         | None                                                                                                                                          | N        |
+| **local_folder**     | Local work-directory path to store the output and to be used as root directory for temporary folders/files. The final output will go into a subdirectory named `prof-${EXEC_ID}` where `exec_id` is an auto-generated unique identifier of the execution. | If the argument is NONE, the default value is the env variable `RAPIDS_USER_TOOLS_OUTPUT_DIRECTORY` if any; or the current working directory. | N        |
+| **jvm_heap_size**    | The maximum heap size of the JVM in gigabytes                                                                                                                                                                                                             | 24                                                                                                                                            | N        |
+| **tools_jar**        | Path to a bundled jar including RAPIDS tool. The path is a local filesystem.                                                                                                                                                                              | Downloads the latest `rapids-4-spark-tools_*.jar` from mvn repo                                                                               | N        |
+| **verbose**          | True or False to enable verbosity to the wrapper script                                                                                                                                                                                                   | False if `RAPIDS_USER_TOOLS_LOG_DEBUG` is not set                                                                                             | N        |
+| **rapids_options**** | A list of valid [Profiling tool options](../../core/docs/spark-profiling-tool.md#qualification-tool-options). Note that (`output-directory`, `auto-tuner`, `combined`) flags are ignored                                                                  | N/A                                                                                                                                           | N        |
+
+If the CLI does not provide an argument `worker_info`, the tool will throw an error and exit.
```

Comment:
This is fine for now.  But would it be simple to allow the user to run profiling without worker_info and then not execute auto-tuner and just generate profiling output files locally?  That would be needed to fully deprecate Profiler java command as some customers have run w/o cluster info to get metrics output without recommendations.

Reply:
If that's the case, then I suggest that to be done within the same PR so we don't have to go back to the `spark_rapids_user_tools` CLI to add features.
The trick with the new `ascli` proposal that it recommends removing the "rapids_options" arguments which means that some arguments consumed by the jar file have to be hardcoded.